recipes/pulsar_phase/gammapy-pint-environment.yml

```diff
@@ -0,0 +1,21 @@
+#Declare your specific environment
+name: pulsar-gammapy
+
+channels:
+  - conda-forge
+
+dependencies:
+  - gammapy=1.0
+  - python=3.9
+  - jupyter
+  - jupyterlab=3.4.8
+  - numpy=1.23
+  - scipy=1.9.1
+  - pandas=1.5.0
+  - matplotlib=3.6.2
+  - astropy=5.1
+  - regions=0.6
+  - iminuit=2.17
```

Comment:
I think gammapy dependencies need not be repeated

Reply:
done